console/frontend/src/main/frontend/src/assets/css/style.css

```diff
@@ -3503,7 +3503,8 @@ body {
   font-size: 13px;
   color: #676a6c;
   overflow-x: hidden;
-  overflow-y: scroll !important;
+  /*overflow-y: scroll !important;*/
```

Comment:
I think you forgot to remove this

Reply:
No I kept it to keep the original file sort of in tact, I don't like making changes in this mess hence why I also have a bootrap-fix.css